grafana-plugin/src/pages/incident/Incident.tsx

```diff
@@ -339,6 +342,25 @@ class IncidentPage extends React.Component<IncidentPageProps, IncidentPageState>
                 />
               </div>
 
+              {Boolean(store.hasFeature(AppFeature.Labels) && incident.labels.length) && (
+                <TooltipBadge
+                  tooltipTitle=""
+                  borderType="secondary"
+                  icon="tag-alt"
+                  addPadding
+                  text={incident.labels.length}
+                  tooltipContent={
+                    <VerticalGroup spacing="sm">
+                      {incident.labels.length
```

Comment:
We have the same check of `length` in line 345. So at this point it will always be truthy which means that `'No labels attached'` will never be displayed. I guess this condition is redundant?